pkg/generator/generator.go

```diff
@@ -57,10 +66,14 @@ func genEmptySet(name string, surveyType SurveyType, opts ...surveyOption) (secr
 	}, nil
 }
 
-func genSurvey(name string, surveyType SurveyType, opts ...surveyOption) (secrets.SourceMap, error) {
-	if surveyType != surveyCredentials && surveyType != surveyParameters {
-		return secrets.SourceMap{}, fmt.Errorf("unsupported survey type: %s", surveyType)
+func formatDescriptionForSurvey(description string) string {
+	if description != "" {
```

Comment:
Any reason for not putting this logic into the `withDescription` function?

Reply:
It's late here and I'm not a smart man... :D 

I wanted it in a separate function so that I could call it from the test files, but agree it would be easier to move it to the withDescription() function. I'll do that now :)